extractor/src/main/java/org/schabi/newpipe/extractor/utils/Utils.java

```diff
@@ -203,4 +204,16 @@ public static boolean isNullOrEmpty(final Collection<?> collection) {
     public static boolean isNullOrEmpty(final Map map) {
         return map == null || map.isEmpty();
     }
-}
\ No newline at end of file
+
+    public static String join(final CharSequence delimiter, final Iterable<? extends CharSequence> elements) {
```

Comment:
Maybe we should replace the duplicate method in the frontend with a call to this one (not in this pr though, I think)
Another note for the future: we should add tests for these utility functions, I think

Reply:
Yes, we should tests these functions. And if there is a duplicate, it should be removed in frontend, not backend. Backend can't call frontend code, whereas the reverse is true.